src/devtools/views/right-rail/index.js

```diff
@@ -17,16 +24,49 @@ class RightRail extends Presenter {
     }
   }
 
+  initDrag = e => {
```

Comment:
I wonder if you could simply always attach the event handlers and never worry about removing them and instead just track an `isDragging` boolean variable that you set to `true` when `mousedown` happens and `false` when `mouseup` happens. This would mean less instances of `addEventListener` and `removeEventListener` calls.

Beyond that I wonder whether there would be any improvement to wrapping the `onDrag` -> `setState` in a `requestAnimationFrame` or whether that would introduce lag time in the UI.